<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DatasetBrowser</class>
 <widget class="QMainWindow" name="DatasetBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dataset Browser</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Select Dataset</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboDataset">
        <property name="currentText">
         <string>MSRDailyActivity3D</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>MSRDailyActivity3D</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>MSRAction3D</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Activities</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listActivities"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnSelectAllActivities">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUnselectAllActivities">
            <property name="text">
             <string>Unselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Actors</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listActors"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnSelectAllActors">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUnselectAllActors">
            <property name="text">
             <string>Unselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Samples</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listSamples"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btnSelectAllSamples">
            <property name="text">
             <string>Select All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnUnselectAllSamples">
            <property name="text">
             <string>Unselect All</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Instances</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listInstances"/>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxSameViewer">
          <property name="text">
           <string>Share last viewer</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxLoop">
          <property name="text">
           <string>Enable loop</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Show Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboType">
            <item>
             <property name="text">
              <string>Depth</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Color</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Skeletal</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
